clarity.d.ts

```diff
@@ -251,7 +254,7 @@ interface IViewportState {
 
 declare const enum Instrumentation {
   JsError,
-  ApiMissing,
+  MissingFeature,
```

Comment:
Should we add a check() to the plug-in to indicate whether the plug-in can active -- i.e.  the APIs needed are supported and other needed plug-ins are present? Similar to what we did in earlier prototypes of this plug-in architecture.